types/task.ts

```diff
@@ -9,6 +15,14 @@ export interface TaskCreationForm {
   tutor: string
 }
 
+export interface TaskUpdateForm {
+  child: string | null
```

Comment:
fait plutot
```suggestion
  child: string | null
```

non ?